docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -23,28 +23,132 @@ Users are able to write their own custom access monitoring queries by querying t
+| `eu-central-1`    | AWS region                                               |
+| `1234567890`      | AWS account ID                                           |
+| `audit-long-term` | S3 bucket used for long-term storage                     |
+| `audit-transient` | S3 bucket used for transient storage                     |
+| `kms_id`          | KMS key ID used for server-side encryption of S3         |
+| `audit_db`        | Glue database used for audit logs                        |
+| `audit_table`     | Glue table used for audit logs                           |
+| `audit_workgroup` | Athena workgroup used for Access Monitoring queries      |
+
+```json
+{
+  "Statement": [
+    {
+      "Action": [
+        "s3:ListBucketVersions",
+        "s3:ListBucketMultipartUploads",
+        "s3:ListBucket",
+        "s3:GetBucketLocation"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:s3:::audit-transient",
+        "arn:aws:s3:::audit-long-term"
+      ]
+    },
+    {
+      "Action": [
+        "s3:PutObject",
+        "s3:GetObjectVersion",
+        "s3:GetObject"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:s3::::audit-transient/results/*",
+        "arn:aws:s3:::audit-long-term/report_results/*"
+      ]
+    },
+    {
+      "Action": [
+        "s3:ListMultipartUploadParts",
+        "s3:GetObjectVersion",
+        "s3:GetObject",
+        "s3:AbortMultipartUpload"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:s3:::audit-transient/results/*",
+        "arn:aws:s3:::audit-long-term/report_results/*",
+        "arn:aws:s3:::audit-long-term/events/*"
+      ]
+    },
+    {
+      "Action": [
+        "glue:GetTable",
+        "athena:StartQueryExecution",
+        "athena:GetQueryResults",
+        "athena:GetQueryExecution"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:glue:eu-central-1:1234567890:table/audit_db/audit_table",
+        "arn:aws:glue:eu-central-1:1234567890:database/audit_db",
+        "arn:aws:glue:eu-central-1:1234567890:catalog",
+        "arn:aws:athena:eu-central-1:1234567890:workgroup/audit_workgroup"
+      ]
+    },
+    {
+      "Action": [
+        "kms:GenerateDataKey",
+        "kms:Decrypt"
+      ],
+      "Effect": "Allow",
+      "Resource": "arn:aws:kms:eu-central-1:1234567890:key/kms_id"
+    }
+  ],
+  "Version": "2012-10-17"
+}
+```
+
+You can used our [Terraform Example](https://github.com/gravitational/teleport/tree/f38bac4bcf6e462a1cab66a0b6074ded4201867d/examples/athena)
+to set up Athena and Access Monitoring AWS resources and generate Athena Backend and Access Monitoring Teleport configuration:
+
+```bash
+$ terraform apply
+...
+ access_monitoring:
+   enabled: true
+   report_results: s3://long-term/report_results
+   role_arn: arn:aws:iam::123456789012:role/AccessMonitoringRole
+   workgroup: access_monitoring_workgroup
+...
+athena_url = "..."
```

Comment:
It is not clear how "access_monitoring" and "athena_url = ..." sections are related to "terraform apply" command. Is this the output from the apply command, or the example config users will need to set?